codeSnippets/snippets/tutorial-full-stack-task-manager/integration-test/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+plugins {
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.kotlinAndroid)
+    alias(libs.plugins.kotlinPluginSerialization)
+}
+
+dependencies {
+    implementation(projects.shared)
+    implementation(projects.server)
+    implementation(projects.composeApp)
+
+    implementation(libs.ktor.server.content.negotiation.jvm)
+    implementation(libs.ktor.client.content.negotiation)
+    implementation(libs.ktor.serialization.kotlinx.json)
+    implementation(libs.androidx.core.ktx)
+
+    testImplementation(libs.ktor.server.test.host)
+    testImplementation(libs.kotlin.test)
+    testImplementation(libs.kotlin.test.junit5)
+
+    testImplementation(libs.junit.jupiter.api)
+    testRuntimeOnly(libs.junit.jupiter.engine)
+}
+
+android {
+    namespace = "io.github.kez_lab.multipatform.full.integrationtest"
+    compileSdk = libs.versions.android.compileSdk.get().toInt()
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+    }
+    defaultConfig {
+        minSdk = libs.versions.android.minSdk.get().toInt()
+    }
+    sourceSets["test"].java.srcDirs("src/test/kotlin")
```

Comment:
Modern versions of AGP add `kotlin/` directory to source sets themself, so we can drop this line